scripts/build-post-list.js

```diff
@@ -58,7 +58,7 @@ function walkDirectories(directories, result, sectionWeight = 0, sectionTitle) {
         details.isIndex = fileName.endsWith('/index.md')
         details.slug = details.isIndex ? sectionSlug : slug.replace(/\.md$/, '')
         if(details.slug.includes('/specifications/') && !details.title) {
-          details.title = capitalize(basename(data.slug))
+          details.title = capitalize(basename(data.slug).slice(1))
```

Comment:
I think that you should first check that `slug` has `v...` prefix, and only then remove it :) You can use `startsWith` (maybe another name of built-in function) function :)

Reply:
Made some changes. Please check if it is fine or not. :)